fx-src/FirefoxStorage.js

```diff
@@ -163,12 +165,14 @@ export default class FirefoxAdapter extends BaseAdapter {
 
     return conn.executeTransaction(function* doExecuteTransaction() {
       // Preload specified records from DB, within transaction.
-      const parameters = {
-        collection_name: collection,
-        // XXX: would be easier if a list of strings could be bound instead.
-        record_ids: options.preload.join("','")
-      };
-      const rows = yield conn.execute(statements.listRecordsById, parameters);
+      const parameters = [
+        collection,
+        ...options.preload
+      ];
+      const placeholders = options.preload.map(_ => "?");
+      const stmt = statements.listRecordsById + "(" + placeholders.join(",") + ");";
```

Comment:
Could you explain me why this is better than the previous version? It genuinely though it was a bad idea to concatenate parameters like that without having an SQL injection escaping (usually done by the client)? I think it was the previous behavior intention.